packages/minimongo/cursor.js

```diff
@@ -25,7 +25,7 @@ export default class Cursor {
 
     this.skip = options.skip || 0;
     this.limit = options.limit;
-    this.fields = options.fields;
+    this.fields = options.projection || options.fields;
```

Comment:
It would require a good deal of changes for a name replacement. For now, I don't think it justifies the cost/benefit as we are still supporting the "fields" field/